### ISSUE TITLE
Instant discount email received twice 

### ISSUE THREAD

**Issue by SumithraKB** [ajency/ProjectX#185]
Verfied the email id once for instant discount, but received the email two times

![image](https://user-images.githubusercontent.com/26400070/33759757-74b19a5a-dc29-11e7-9040-045d68bd922c.png)

**Comment by tanvisalkar** [comment]
unable to replicate the issue... possibility: the button was clicked more than once..

**Comment by cinthiavernekar** [comment]
@tanvisalkar Yes the issue occurs when user clicks on submit button more than once.
This needs to be handled. May be we can disable the submit button once clicked.
